wsd/Admin.cpp

```diff
@@ -603,42 +603,32 @@ unsigned Admin::getNetStatsInterval()
 
 std::string Admin::getChannelLogLevels()
 {
-    std::string result;
-    // Get the list of channels..
-    std::vector<std::string> nameList;
-    Log::logger().names(nameList);
+    unsigned int wsdLogLevel = Log::logger().get("wsd").getLevel();
+    std::string result = "wsd=" + levelList[wsdLogLevel];
 
-    std::string levelList[9] = {"none", "fatal", "critical", "error", "warning", "notice", "information", "debug", "trace"};
-
-    for (size_t i = 0; i < nameList.size(); i++)
-    {
-        result += (nameList[i] != "" ? nameList[i]: "?") + '=' + levelList[Log::logger().get(nameList[i]).getLevel()] + (i != nameList.size() - 1 ? " ": "");
-    }
+    result += " kit=" + (_forkitLogLevel.empty() != true ? _forkitLogLevel: levelList[wsdLogLevel]);
 
     return result;
 }
 
-void Admin::setChannelLogLevel(const std::string& _channelName, std::string _level)
+void Admin::setChannelLogLevel(const std::string& _channelName, std::string level)
```

Comment:
Good, `_level` should be `level`, perhaps do the same with `_channelName` ?

Reply:
Thanks for the notes. I replaced _channelName variable.